### ISSUE TITLE
New project request - webios

### ISSUE THREAD

**Issue by upintheairsheep** [yume-chan/ya-webadb#391]
i use android for my phone but I use an iPad too. If you have an iOS device reply back. If you don’t continue working on ya-webadb. I’m mostly excited about the backup system to backup, restore, and view .ab backups.

**Comment by upintheairsheep** [comment]
I use iMazing on my pc to manage my iPad sometimes and they have simple options like backup for small backups, simple power controls, viewing device info, file browser, etc. If you have an iOS device say that you do. Let’s reverse engineer iMazing with WireShark!

**Comment by hacknorris-aka-penguin** [comment]
do you know 3utools ? wasy better app for desktop... i also use 2 ipads...

**Comment by yume-chan** [comment]
Interesting idea, I did some exploration.

On macOS and Linux, it's possible to use WebUSB API to interact with iOS device. However, on Windows, iTunes is still required, and Direct Sockets API must be used to connect to iTunes (more precisely, Apple Mobile Device Service).

I don't want to create a project that's not portable and uses experimental API that may never be stabilized. I don't have time to do that either.

Closing as out of scope.
